www/index.html

```diff
@@ -44,29 +44,17 @@ <h3 class="section">File manager</h3>
                                             title="Start upload to current dir">Upload</label>
                     <input type="file" id="file-manager-file" style="display:none" onchange="uploadFile()"></div>
                 <div class="section">
-
                     <table>
                         <thead>
                         <tr>
-                            <th>name</th>
-                            <th>size</th>
+                            <th>NAME</th>
+                            <th>SIZE</th>
```

Comment:
Sorry for the misunderstanding, I meant only the first letter, i.e. `Name` and `Size`.